### PR TITLE
C2 - added README.md file

### PR DESCRIPTION
**Pull request by DhirajKelhe**
README.md file is necessary and your project is missing that file. So, here you go..

### CODE REVIEW

**Comment by AIMasochists** [comment]
Thanks for contribution. Will check that later.

**Comment by DhirajKelhe** [comment]
Ok

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This project is for calculating cube of a number.
+
+============
+## Examples:
+* cube(3) will return 27
+* cube(1) will return 1
```
Comment (AIMasochists) [inline README.md]: Explain these examples.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+* cube(0) will return 0
+* cube(10) will return 1000
```
Comment (AIMasochists) [inline README.md]: Kindly explain these examples.

Comment (DhirajKelhe) [reply README.md]: Ok. Wait a moment.

**Comment by DhirajKelhe** [comment]
Everything is done.

**Comment by AIMasochists** [comment]
nit

**Pull request merged**